tools/Evm/Evm/t8n/JsonTypes/TransactionJson.cs

```diff
@@ -0,0 +1,81 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Ethereum.Test.Base;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Core.Eip2930;
+using Nethermind.Core.Test.Builders;
+using Nethermind.Crypto;
+using Nethermind.Int256;
+
+namespace Evm.t8n.JsonTypes;
+
+public class TransactionJson
```

Comment:
Does this differ substantially from `TransactionForRPC` or maybe it can be reused here?